In-house patch to remove unnecessary dependencies from Cinder's
requirements files. The specific reasons are as follows:

kombu		Not applicable

oslo.rootwrap	Not applicable to Solaris

paramiko	Not applicable to Solaris (various drivers specific)

pycrypto	Not applicable to Solaris (various drivers specific)

rtslib-fb	Not applicable to Solaris (Linux iSCSI specific)

--- cinder-2014.2.2/cinder.egg-info/requires.txt.~1~	2015-02-05 08:05:50.000000000 -0800
+++ cinder-2014.2.2/cinder.egg-info/requires.txt	2015-02-23 13:49:52.736413251 -0800
@@ -6,18 +6,14 @@ eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
 iso8601>=0.1.9
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
-oslo.rootwrap>=1.3.0
 osprofiler>=0.3.0                       # Apache-2.0
-paramiko>=1.13.0
 Paste
 PasteDeploy>=1.5.0
-pycrypto>=2.6
 python-barbicanclient>=2.1.0,!=3.0.0,<3.0.2
 python-glanceclient>=0.14.0
 python-novaclient>=2.18.0
@@ -25,7 +21,6 @@ python-swiftclient>=2.2.0
 requests>=1.2.1,!=2.4.0
 Routes>=1.12.3,!=2.0
 taskflow>=0.4,<0.7.0
-rtslib-fb>=2.1.39
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
 sqlalchemy-migrate==0.9.1
--- cinder-2014.2.2/requirements.txt.~1~	2015-02-05 08:03:26.000000000 -0800
+++ cinder-2014.2.2/requirements.txt	2015-02-23 13:52:20.137709426 -0800
@@ -10,18 +10,14 @@ eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
 iso8601>=0.1.9
 keystonemiddleware>=1.0.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config>=1.4.0  # Apache-2.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
-oslo.rootwrap>=1.3.0
 osprofiler>=0.3.0                       # Apache-2.0
-paramiko>=1.13.0
 Paste
 PasteDeploy>=1.5.0
-pycrypto>=2.6
 python-barbicanclient>=2.1.0,!=3.0.0,<3.0.2
 python-glanceclient>=0.14.0
 python-novaclient>=2.18.0
@@ -29,7 +25,6 @@ python-swiftclient>=2.2.0
 requests>=1.2.1,!=2.4.0
 Routes>=1.12.3,!=2.0
 taskflow>=0.4,<0.7.0
-rtslib-fb>=2.1.39
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
 sqlalchemy-migrate==0.9.1
